package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-autoql",
-  "version": "8.8.7",
+  "version": "8.8.8",
```

Comment:
We had version 8.8.8 just now.
Should update to 8.8.9

Reply:
updated to 8.8.9